tests/templates/test_subroutines/test_kupccgsd.py

```diff
@@ -54,7 +54,8 @@ def test_standard_validity(k, delta_sz, init_state, wires):
 
     op = qml.kUpCCGSD(weights, wires=wires, k=k, delta_sz=delta_sz, init_state=init_state)
 
-    qml.ops.functions.assert_valid(op)
+    skip_diff = k > 1
```

Comment:
So that we have at least one test with `k > 1` that tests differentiability, maybe we can change the condition to the following?
```suggestion
    skip_diff = len(wires) > 4
```

Let me know what you think 😄

Reply:
Looking at the durations artifact, the test with `k=2` and 4 wires takes 37 seconds. While still an expensive test, it's way better than the 233 and 405 seconds taken respectively by the two larger test cases 😅

Reply:
Yeah, that's actually what I had first :D I'm happy to revert to `len(wires)>4` :)